### PR TITLE
small fix in export script

### PR DESCRIPTION
**Pull request by michielbdejong**
* [ ] Are you working on the latest release?
* [ ] Have you tested it locally?
* [ ] Please explain your change

Thanks !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Please refer to the CONTRIBUTING.md guide for more information. If anything is m
 
 ## Database
 
-All the details on the database schema can be found on the [wiki](https://github.com/tosdr/phoenix/wiki/database).
+All the details on the database schema can be found on the [wiki](https://github.com/tosdr/edit-tosdr-org/wiki/database).
```
Comment (Vinnl) [inline README.md]: So apparently you did a rename - but the URL of the current page for me says `edit.tosdr.org`, not `edit-tosdr-org`...

File: README.md [hunk 2]
```diff
@@ -21,31 +21,31 @@ be kept secret. Never commit a database dump to git, or share it with someone wh
...
 ```sh
-# make sure you have phoenix checked out next to tosdr-build in a folder:
-git clone https://github.com/tosdr/phoenix
-git clone https://github.com/tosdr/tosdr-build
-mkdir tosdr-build/src/pointsPhoenix
-cd phoenix
+# make sure you have edit-tosdr-org checked out next to tosdr.org in a folder:
+git clone https://github.com/tosdr/edit-tosdr-org
+git clone https://github.com/tosdr/tosdr.org
+mkdir tosdr.org/src/pointsPhoenix
```
Comment (Vinnl) [inline README.md]: I guess the rename of `tosdr-build` to `tosdr.org` still has to happen? (As in: the GitHub interface still lists `tosdr.org` as the project containing the build output.)

File: README.md (hunk 1 above)
Comment (michielbdejong) [reply README.md]: Ah good catch!

File: README.md (hunk 2 above)
Comment (michielbdejong) [reply README.md]: It happened see https://github.com/tosdr/tosdr-build#moved

**Comment by JesseWeinstein** [comment]
I think it's safe to merge this?

**Pull request merged**